File: 25.07.2024/Library Management System.c
#include <stdio.h>
#include <stdbool.h>
struct Book {
    int id;
    char title[100];
    char author[100];
    bool isBorrowed;
};
void borrowBook(struct Book *book) {
    if (!book->isBorrowed) {
        book->isBorrowed = true;
        printf("Book \"%s\" borrowed successfully!\n", book->title);
    } else {
        printf("Sorry, this book is already borrowed.\n");
    }
}
int main() {
    struct Book myBook = {1, "The Great Gatsby", "F. Scott Fitzgerald", false};
    borrowBook(&myBook);
    return 0;
}

File: 25.07.2024/Library Management System.java
package net.javaguides.lms;
public class Book {
    private int id;
    private String title;
    private String author;
    private boolean isBorrowed;
    public void borrowBook() {
        if (!isBorrowed) {
            isBorrowed = true;
            System.out.println("Book \"" + title + "\" borrowed successfully!");
        } else {
            System.out.println("Sorry, this book is already borrowed.");
        }
    }
    public static void main(String[] args) {
        Book myBook = new Book(1, "The Great Gatsby", "F. Scott Fitzgerald");
        myBook.borrowBook();
    }
}
